plugin/path_config.go

```diff
@@ -106,23 +136,25 @@ func (b *GcpAuthBackend) pathConfigRead(ctx context.Context, req *logical.Reques
 	if err != nil {
 		return nil, err
 	}
-	if config == nil {
```

Comment:
For this location, the effect of the config never being nil is desirable because now reads when the config is unset will result in the user learning their default alias, and their default auth metadata.